addons/arsenal/functions/fnc_onArsenalClose.sqf

```diff
@@ -70,6 +70,9 @@ if (!isNil QGVAR(moduleUsed)) then {
     objNull remoteControl GVAR(center);
 };
 
+// Update player unique items cache
+ACE_player call EFUNC(common,uniqueItems);
+
```

Comment:
Does the "loadout" player event handler not handle this?

Reply:
Only clears the variable.

Reply:
I don't see why arsenal needs to update the cache. It will automatically updated next time the getter function is called.

Reply:
Doesn't *need* to, this is just preemptively doing it since exiting the arsenal has a milisecond or two of downtime. I don't particularly care either way for it, can be deleted if needed.